src/main/java/org/codehaus/groovy/runtime/ArrayUtil.java

```diff
@@ -61,31 +55,32 @@
 public class ArrayUtil {
     private static final Object[] EMPTY = new Object[0];
 
-    public static MethodHandle getCloneArrayMethodHandle() {
```

Comment:
`getCloneArrayMethodHandle` was added in 4.0.5-SNAPSHOT, so removing it will not break binary compatibility